.github/workflows/ci.yml

```diff
@@ -148,6 +148,33 @@ jobs:
           acl_provider_endpoint: ${{ secrets.ACL_PROVIDER_ENDPOINT }}
 
   e2e:
+    needs: prepare
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Restore cache
+        uses: ./.github/actions/restore-cache
+
+      - name: Set up Node
+        uses: ./.github/actions/setup-node
+
+      - name: Set up Playwright browsers
+        run: yarn test:e2e:install
```

Comment:
No point for me, it's not a package we publish and it's not a projet. So we don't need all of `types`, `unit` for example.

Reply:
Even audit is not crazy useful for us, as it's just a dev project for test. We should maintain or deps for usage not because of a vuln.

Reply:
I finally add it, since now we have also a lint and types scrit that can be cool to be runned in CI since it can find problems.

I added the unnecessary script "audit" & "unit" just so it doesn't break. At least I like the fact we are consistent in our package when possible. Let me know what you think.